src/services/functionAppService.ts

```diff
@@ -43,63 +44,45 @@ export class FunctionAppService extends BaseService {
     return response.data.value;
   }
 
-  public async deleteFunction(functionName) {
-    this.serverless.cli.log(`-> Deleting function: ${functionName}`);
-    return await this.webClient.webApps.deleteFunction(this.resourceGroup, this.serviceName, functionName);
-  }
-
-  public async syncTriggers(functionApp) {
-    this.serverless.cli.log('Syncing function triggers');
-
-    const syncTriggersUrl = `${this.baseUrl}${functionApp.id}/syncfunctiontriggers?api-version=2016-08-01`;
-    await this.sendApiRequest('POST', syncTriggersUrl);
-  }
+  async zipDeploy(functionApp) {
+    const functionAppName = functionApp.name;
+    this.serverless.cli.log(`Deploying zip file to function app: ${functionAppName}`);
 
-  public async cleanUp(functionApp) {
-    this.serverless.cli.log('Cleaning up existing functions');
-    const deleteTasks = [];
-
-    const serviceFunctions = this.serverless.service.getAllFunctions();
-    const deployedFunctions = await this.listFunctions(functionApp);
-
-    deployedFunctions.forEach((func) => {
-      if (serviceFunctions.includes(func.name)) {
-        this.serverless.cli.log(`-> Deleting function '${func.name}'`);
-        deleteTasks.push(this.deleteFunction(func.name));
-      }
-    });
-
-    return await Promise.all(deleteTasks);
-  }
-
-  public async listFunctions(functionApp) {
-    const getTokenUrl = `${this.baseUrl}${functionApp.id}/functions?api-version=2016-08-01`;
-    const response = await this.sendApiRequest('GET', getTokenUrl);
-
-    return response.data.value || [];
-  }
-
-  public async uploadFunctions(functionApp): Promise<any> {
```

Comment:
I know we are using `zipDeploy` method for the service deployment - but I think we should keep a more generic name as the method name.  Later on when we introduce `runFromPackage` we will have multiple branches within this method.  Let's stick with `uploadFunctions` for now.

Reply:
just to be clear, I am adding `runFromPackage` in this PR already, it's the change in the arm template